src/widgets/source/ui/CalibratorsPlotWidget.cpp

```diff
@@ -57,39 +57,37 @@ namespace SmartPeak
     }
   }
 
-  void CalibratorsPlotWidget::recomputeCalibration()
+  void CalibratorsPlotWidget::recomputeFitCalibration(const std::string& component_name)
```

Comment:
I believe the "sequence_segment_name" would be needed here as well?  Or is that not correct because I see an internal variable on line 71 below called "sequence_segment_name_" that is specified before the call to ``fit_calibration.process()`?

Reply:
Only the component name is actually needed. it may not be easy to see in the code there.

when multiple curves are displayed on the plot. we used the component selected on the left panel. but it has to be the component of the point selected by the user.
However, about the sequence segment, all the curves from the plot belong to the same sequence segment. there is no ambiguity. so we can stay using the sequence segment selected on the left panel.
i think it's ok, but we can say it's a bit disturbing, i can add that parameter, but i will just pass the value of the left panel there.